### ISSUE TITLE
TextArea component is missing autoCompleteType

### ISSUE THREAD

**Issue by elenitaex5** [GeekyAnts/NativeBase#5438]
### Description

When I include TextArea component without any props as `<TextArea />` it includes a Typescript error, missing `autoCompleteType`

### CodeSandbox/Snack link

https://codesandbox.io/s/heuristic-david-cdf8zi

### Steps to reproduce

Include <TextArea /> component in a project having native-base, typescript and eslint.

### NativeBase Version

^3.4.15

### Platform

- [ ] Android
- [ ] CRA
- [ ] Expo
- [ ] iOS
- [ ] Next

### Other Platform

The error is shown in IDE.

### Additional Information

<img width="984" alt="image" src="https://user-images.githubusercontent.com/14013852/193532207-d83137dc-1456-4ecb-b032-206356d85967.png">

**Comment by ankit-tailor** [comment]
Hey @elenitaex5, Thanks for reporting the issue. We'll look into it.

**Comment by alalfakawma** [comment]
Any updates on this issue?

**Comment by elenitaex5** [comment]
@alalfakawma I don't think so... If it would be fix it, the issue would be closed by authors or collabs.

**Comment by alalfakawma** [comment]
> @alalfakawma I don't think so... If it would be fix it, the issue would be closed by authors or collabs.

Yeah, true. Let's hope this gets fixed.

**Comment by blakew** [comment]
I'm also experience this issue

**Comment by quentinmassias** [comment]
Same error here

**Comment by arcticfly** [comment]
Same issue

**Comment by swissjake** [comment]
<TextArea
              autoCompleteType={true}
              mt="2"
              bgColor={"white"}
              rounded="10px"
/>

autoCompleteType={true} clears the error

**Comment by Justin-Garey** [comment]
I'm having the same issue. I'll just be adding the prop and setting to false for now.

**Comment by ricohumme** [comment]
> <TextArea autoCompleteType={true} mt="2" bgColor={"white"} rounded="10px" /> autoCompleteType={true} clears the error

Nowhere in the code does indicates that this value is of a boolean type?
For me, just defining the property resolves the issue for TS validity, but could just as easy be null or an empty string.
@ankit-tailor can you elaborate on what the type is for autoCompleteType ?

**Comment by ricohumme** [comment]
> > <TextArea autoCompleteType={true} mt="2" bgColor={"white"} rounded="10px" /> autoCompleteType={true} clears the error
> 
> Nowhere in the code does indicates that this value is of a boolean type? For me, just defining the property resolves the issue for TS validity, but could just as easy be null or an empty string. @ankit-tailor can you elaborate on what the type is for autoCompleteType ?

Edit: Should be of type string so it seems

**Comment by saoudnazir** [comment]
Same issue

**Comment by levietquang230996** [comment]
thank you so much . I fixed

**Comment by levietquang230996** [comment]
> <TextArea autoCompleteType={true} mt="2" bgColor={"white"} rounded="10px" /> autoCompleteType={true} clears the error

thank you so much . I fixed
